apps/admin_panel/assets/src/omg-wallet/walletProvider.js

```diff
@@ -8,12 +8,18 @@ class WalletProvider extends Component {
     render: PropTypes.func,
     walletAddress: PropTypes.string,
     getWalletById: PropTypes.func,
-    wallet: PropTypes.object
+    wallet: PropTypes.object,
+    pooling: PropTypes.bool
   }
 
   componentDidMount = () => {
     if (!this.props.wallet) {
```

Comment:
Would there be a case when the `componentDidMount` is called and `this.props.wallet` is already set to something?

Reply:
@T-Dnzt yes, if you came from wallets page ( witch will fetch multiple wallets ) this case wallet will already exists and will not be fetch again.

Reply:
So if that's the case, you wouldn't set up the auto-reload every 5 seconds, right? Isn't that something we want no matter what?

Reply:
Oh you just fixed it, cool.